research/src/pages/skeleton.proposal.mdx

```diff
@@ -0,0 +1,53 @@
+---
+menu: Components
+name: Skeleton (Editor's Draft)
+path: /components/skeleton
+pathToResearch: /components/skeleton.research
+---
+
+## Overview
+
+Displays a placeholder reserving the place for the content that is being loaded or used to prototype layouts.
+
+## Use Cases
+
+The Skeleton component is primarily leveraged to reserve a place in the layout for a content that will be shown asynchronous avoiding elements moving to adapt to new dimensions.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/skeleton/)
+- [Semantic UI](https://semantic-ui.com/elements/placeholder.html)
+- [Fabric UI](https://developer.microsoft.com/en-us/fluentui#/controls/web/shimmer)
+
+## &lt;skeleton&gt; Properties <a href="#skeleton-properties" id="skeleton-properties"></a>
```

Comment:
Any reference to `skeleton` needs to be changed to `oui-skeleton` per our resolution since web components require the dash. Granted, if any of these make their way into the platform we expect them to not have a dash.